### PR TITLE
Change list of providers to multiselect

### LINKED ISSUE #96: Change list of providers to multiselect
Related with #76 

> the list of providers should be a multi-select, rather than either a specific one or all of them

### PR DESCRIPTION
**Pull request by funcionante**
Resolves #96.

### CODE REVIEW

File: dicoogle/src/main/resources/webapp/index-template.html [hunk 1]
```diff
@@ -30,14 +30,17 @@
 
     <link rel="stylesheet" href="css/jquery-ui.css">
 
-     <!--Custom JS -->
-     <!--
+    <!-- React Select CSS -->
+    <link rel="stylesheet" href="https://unpkg.com/react-select/dist/react-select.css">
```
Comment (Enet4) [inline dicoogle/src/main/resources/webapp/index-template.html]: We prefer keeping these resources locally. Can you add this css file to our css/ folder and import it from there?

**Comment by bastiao** [comment]
Nice change. We hope that react-select release a final/stable version soon (instead of RC - release candidate).

**Comment by bastiao** [comment]
react-select has now final releases. It might be interested to upgrade.
https://github.com/JedWatson/react-select/releases

**Comment by funcionante** [comment]
@bastiao, now it is upgraded.

**Pull request merged**